src/examples/turtlebot4/README.md

```diff
@@ -0,0 +1,138 @@
+
+   1. Create `whoami` package for turtlebot4 in `src/examples/turtlebot4`
+
+      ```bash
+      . ./setup_shell.sh
+      poetry run create_rai_ws --name turtlebot4 --destination-directory src/examples
+      ```
+
+   2. Download official turtlebot4 [data sheet](https://bit.ly/3KCp3Du) into
+      `src/examples/turtlebot4_whoami/description/documentation`
+   3. Download [image](https://s3.amazonaws.com/assets.clearpathrobotics.com/wp-content/uploads/2022/03/16113604/Turtlebot-4-20220207.44.png) of turtlebot4 into `src/examples/turtlebot4_whoami/description/images`
+   4. Create robot's identity. Run the `parse_whoami_package`. This will process the documentation, building
+      it into a vector database, which is used by RAI agent to reason about its identity.
+
+      > **NOTE**: Vector database is created using the OpenAI API. Parsing bigger documents
+      > might lead to costs. Embedding model can be configured in
+      > [config.toml](https://github.com/RobotecAI/rai/blob/development/config.toml#L13)
+
+      ```bash
+      poetry run parse_whoami_package src/examples/turtlebot4_whoami/description
+      # you will be asked to press `y` to continue
+      ```
+
+   5. Rebuild the workspace
+      ```bash
+      colcon build --symlink-install
+      ```
+
+   Congratulations! Your `rai_whoami_node` is configured. In the follwing steps
+   your RAI agent will have turtlebot4 personality.
+
+5. Run rai agent:
+
+   ```bash
+   ros2 launch ./src/examples/turtlebot4/turtlebot.launch.xml \
+       game_launcher:=./src/examples/turtlebot4/simulation/TurtleBot4DemoGamePackage/TurtleBot4Demo.GameLauncher
+   ```
+
+6. Open you internet browser and go to `localhost:8501`
+7. You can interact with your RAI agent through the chat. On the left you can interact
+   with the RAI HMI Agent. On the right you will see status of missions that were send
+   to the RAI Node and are executed on your robot.
+
+   - `HMI Agent` is responsible for human-robot interaction.
+   - `RAI Node` is responsible for executing tasks on the robot.
+
+   For example you can try such prompt:
+
+   - Are you able to bring you something from the kitchen? (testing robot's identity)
+   - What are your ros2 interfaces? (discovery of ros2 interfaces)
+   - tell me what you see (interpretation of camera image)
+   - Drive towards the chair (when table is not visible, robot rejects task that it cannot do)
+   - Spin yourself left by 45 degrees (interaction with the robot using ros2 interfaces) - table with the robotic are shoud be visible in the camera
+   - Use robotic arm to pick up a box from the table (identity and intefaces doesn't allow it)
+   - Drive towards the table (when table is visible, testing ability to interpret camera image and perform actions based on the knowledge)
+   - Drive behind the table (likely will fail in current `rai` version due to limitations in spatial reasoning)
+
+### Troubleshooting
+
+_My robot doesn't have the identity._
```

Comment:
```suggestion
_My robot doesn't have an identity._
```

Reply:
Also add how this would look like for the user, e.g seeing an error message etc.